subprojects/docs/src/docs/release/notes.md

```diff
@@ -164,6 +164,22 @@ On case-insensitive file systems (e.g. NTFS and APFS), a file/folder rename wher
 For example, renaming an input of a `Copy` task called `file.txt` to `FILE.txt` will now cause `FILE.txt` being created in the destination directory. 
 The `Sync` task and `Project.copy()` and `sync()` operations now also handle case-renames as expected.
 
+## Unavailable files are handled more gracefully
+Generally, broken symlinks, named pipes and unreadable files/directories (hereinafter referred to as unavailable files)
+found in inputs/outputs of tasks are handled gracefully from now on: similarly to missing files.
```

Comment:
We should try to use one line per sentence, so it is easier to comment and track changes.

Reply:
That's what I tried, breaking longer sentences to multiple lines, it's what I saw in other places.
Do you suggest not breaking long sentences to multiple lines?

Reply:
Done

Reply:
That is what I am suggesting. The user manual (or release notes) are not consistent, yet. We try to use the one sentence per line style when we touch or create something new, though.